study.md

```diff
@@ -66,15 +90,15 @@ nearest_star = the_sun
 Will modifying `sol` affect `the_sun`? Why or why not?
 
 ```md
-<!-- your answer here -->
+Modifying sol will not affect the_sun.  This is because they were declared independently and no relationship was formed between sol and the_sun.
 ```
 
 ## Ruby Object Equality: Part II
 
 Will modifying `the_sun` affect `nearest_star`? Why or why not?
 
 ```md
-<!-- your answer here -->
+No.  Modifying the_sun will not affect nearest_star because when redeclaring a value of the_sun (eg. Star.new(0,2)), it breaks the relationship between nearest_star and the_sun.
```

Comment:
`the_sun` and `nearest_star` are both references to the same object so any modification using one identifier will have its changes reflected in the other.